datahub-web-react/src/app/analytics/analytics.ts

```diff
@@ -7,15 +7,25 @@ import { getBrowserId } from '../browserId';
 
 const appName = 'datahub-react';
 
+export const THIRD_PARTY_LOGGING_KEY = 'enableThirdPartyLogging';
+
 const analytics = Analytics({
     app: appName,
     plugins: plugins.filter((plugin) => plugin.isEnabled).map((plugin) => plugin.plugin),
 });
 
 const { NODE_ENV } = process.env;
 
+export function getMergedTrackingOptions(options?: any) {
+    return {
+        ...options,
+        plugins: {
+            mixpanel: JSON.parse(localStorage.getItem(THIRD_PARTY_LOGGING_KEY) || 'false'),
```

Comment:
based on the mixpanel configuration block i see here - https://github.com/datahub-project/datahub/blob/master/datahub-web-react/src/conf/analytics.ts#L6

it seems like it requires a structure like

```
mixpanel: {
   token: <token>
}
```

Reply:
talked irl - passing in false here will disable logging for the provider we pass in for this tracking call. The tokens are still stored the same way they were before and still being passed into the `Analytics` object. Here's the docs for selectively disabling plugins (see "Selectively disabling plugins" lol) https://getanalytics.io/plugins